admin/package.json

```diff
@@ -13,6 +13,7 @@
     "jquery-steps": "1.1.0",
     "jquery-treegrid": "0.3.0",
     "jquery.cookie": "1.4.1",
+    "magicsuggest-alpine": "1.0.0",
```

Comment:
Is it a good choice?
in ngeo we use corejs-typeahead who has about 12000 monthly download and this one has only 16 monthly download :-/ 
or do he has an other goal?

Reply:
Can be discussed, I do not know corejs-typeahead very well.

Reply:
Last time I have used typeahead, interface was moving in each release.

Reply:
Not personally but at camptocamp we have a good knowledge on this library.
The API didn't change the last 3 years...

Reply:
I would like to merge, for now, it is working with magicsuggest, this could be changed later.